src/SFA.DAS.ApplyService.Application.UnitTests/Handlers/InProgressAssessorApplicationsHandlerTests/InProgressAssessorApplicationsHandlerTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApplyService.Application.Apply;
+using SFA.DAS.ApplyService.Application.Apply.Assessor;
+using SFA.DAS.ApplyService.Domain.Apply;
+
+namespace SFA.DAS.ApplyService.Application.UnitTests.Handlers.InProgressAssessorApplicationsHandlerTests
+{
+    [TestFixture]
+    public class InProgressAssessorApplicationsHandlerTests
+    {
+        private Mock<IAssessorRepository> _repository;
+        private InProgressAssessorApplicationsHandler _handler;
+
+        [SetUp]
+        public void TestSetup()
+        {
+            _repository = new Mock<IAssessorRepository>();
+            _handler = new InProgressAssessorApplicationsHandler(_repository.Object);
+        }
+
+        [Test]
+        public async Task Get_in_progress_applications_returns_in_progress_applications_for_the_user()
+        {
+            var expectedUser = "sadjkffgdji";
+            var expectedResult = new List<RoatpAssessorApplicationSummary>();
+            _repository.Setup(x => x.GetInProgressAssessorApplications(expectedUser)).ReturnsAsync(expectedResult);
```

Comment:
Can you add at least one item to this list and test it's in the actual?

Reply:
It's comparing the instance returned from the method is the same as what is returned from the mock, adding an item to the list won't make the blindest bit of difference.